doc/api/url.md

```diff
@@ -948,6 +948,20 @@ changes:
 The `url.format()` method returns a formatted URL string derived from
 `urlObject`.
 
+```js
+url.format({
+  protocol: 'https',
+  hostname: `example.com`,
+  pathname: '/some/path',
+  query: {
+    page: 1,
+    format: 'json'
+  }
+});
+
+// => 'https://example.com/some/path?page=1&format=json'
```

Comment:
Other places in the file use `Prints` or just show the output below. I personally think a "Prints" would be better than `=>`.

Reply:
I did a search in the docs and found several instances of `// => ` symbolizing a returned value. Also, this method doesn't actually print a value, it just returns it.

Reply:
I would be open to just doing `// => 'https://...`. That was another "convention" I found when looking at existing code examples in Node's docs.

Reply:
In this case I only spoke about this specific document. And yes, `// =>` is still used some times but it is not really common as far as I know.

So I personally would say using `// 'https:...` would be good.